vc_vidyo/indico_vc_vidyo/plugin.py

```diff
@@ -203,8 +203,10 @@ def create_room(self, vc_room, event):
 
             else:
                 # get room back, in order to fetch Vidyo-set parameters
-                created_room = client.find_room(extension)[0]
+                created_room = client.find_room(extension)
 
+                if not created_room:
+                    raise VCRoomNotFoundError(_("This room doesnt exist in Vidyo"))
```

Comment:
`doesn't` or `does not`

Also, I think the error message is not really clear. I'd use something like "Could not find newly created room in Vidyo".